tools/setup-test-users.sh

```diff
@@ -2,7 +2,7 @@
 
 set -e
 ## This script sets up users for running CLI tests. It only needs to be run once per SAM instance.
-## Keep this script in sync with the harness.TestUsers class in the src/test/java directory.
+## Keep this script in sync with test users in your testconfig file (eg `testconfig/broad.json`).
```

Comment:
Could this script take in the test config as input? If so, it's fine to do in a follow-on PR, but here can you please: add a todo to point to a ticket and rename this script e.g. `setup-broad-test-uesrs.sh`?

Reply:
Added TODO, thanks.